sensor_msgs/PointCloud2 cloud_msg
float32 non_planar_ratio
float32 cluster_tolerance
---
Cluster[] clusters
Plane[] planes